src/scripts/main.js

```diff
@@ -1 +1,84 @@
 'use strict';
+
+const body = document.querySelector('body');
+const successBothClick = { left: false, right: false };
+
+const firstPromise = new Promise((resolve, reject) => {
+  body.addEventListener(
+    'click',
+    (e) => {
+      if (e.clientX <= body.clientWidth / 2) {
```

Comment:
left and right in task description means left or right mouse buttons ('click' or 'contextmenu' events).
no need to count body clientWidth